zap/src/main/java/org/zaproxy/zap/spider/parser/SpiderHtmlParser.java

```diff
@@ -41,13 +44,21 @@ public class SpiderHtmlParser extends SpiderParser {
 
     /** The Constant URL_PATTERN defining the pattern for a meta url. */
     static final Pattern URL_PATTERN =
-            Pattern.compile("url\\s*=\\s*[\"']?([^;'\"]+)", Pattern.CASE_INSENSITIVE);
+            Pattern.compile(
+                    "(?:url\\s*=|report-uri)\\s*[\"']?([^;'\"]+)", Pattern.CASE_INSENSITIVE);
 
     private static final Pattern PLAIN_COMMENTS_URL_PATTERN =
             Pattern.compile(
-                    "(?:http(?:s?):)?//[^\\x00-\\x1f\"'\\s<>#()\\[\\]{}]+",
+                    "(?:http(?:s?):)?/?/[^\\x00-\\x1f\"'\\s<>#()\\[\\]{}]+",
                     Pattern.CASE_INSENSITIVE);
 
+    /** The Constant log. */
+    private static final Logger log = LogManager.getLogger(SpiderHtmlParser.class);
```

Comment:
Should be removed.

Reply:
Yes, I forgot to remove it, added for debugging purposes. Now removed :)

Reply:
Still present

Reply:
Added it back last time for debugging, removed it again.